packages/shared/src/variants/quantum.ts

```diff
@@ -23,7 +23,7 @@ class BadukHelper {
     this.badukGame.playMove(player, "pass");
   }
 
-  play(player: number, move: string): Coordinate[] {
```

Comment:
Nit: I like the rule ["Return the most specific type, accept the most generic type"](https://enterprisecraftsmanship.com/posts/return-the-most-specific-type/).  In other words, I usually accept `CoordinateLike` as argument, and return `Coordinate`.  That way, callers have access to all the utility methods of the `Coordinate` class, but only need to pass the bare minimum (`{x: 1, y: 2}`) as a parameter.

Reply:
Changed the types in quantum.ts thusly